3-appfactory/modules/app-group-baseline/main.tf

```
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  cloudbuild_sa_roles = { for env in keys(var.envs) : env => {
+    project_id = module.app_env_project[env].project_id
+    roles      = var.cloudbuild_sa_roles[env].roles
+  } }
+}
+
+// Create admin project
+module "app_admin_project" {
+  source            = "terraform-google-modules/project-factory/google"
+  version           = "11.3.0"
```

Any reason not to use the current version?
```suggestion
  version           = "~> 14.5"
```